src/domain/project/project.interface.ts

```diff
@@ -7,7 +7,7 @@ export interface IProject {
   textID: string;
   name: string;
   description?: string;
-  state: string;
+  state?: string;
```

Comment:
state should be a required field, certainly moving forward - so suggest not to make it optional now